packages/store/src/tightcoder/DecodeSlice.ejs

```diff
@@ -0,0 +1,81 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.8.0;
+
```

Comment:
Should we put a comment on the top of the file saying something like "Autogenerated from DecodeSlice.ejs via yarn codegen, do not edit manually"?

Reply:
> Also, do you think this PR would be a good place to rename `TightCoder` into `Serializer` (as proposed by @holic in [#387 (comment)](https://github.com/latticexyz/mud/pull/387#discussion_r1106941413))?

I'd rather do such encompassing renamings closer to v2 completion when we have world and autogen merged, to avoid changing WIP branches often. Will give more time to think about the name too

Reply:
Added one, see if it's ok. I'm not sure we need the comment customized to filename, but mentioning which file shouldn't be edited seems important
(for schema autogen I just write `/* Autogenerated file. Do not edit manually. */` but since here both files are together, it's not very clear)